miio/roidmivacuum_miot.py

```diff
@@ -0,0 +1,702 @@
+                for prop in self.get_properties_for_mapping(max_properties=10)
+            }
+        )
+
+    @command()
+    def cleaning_summary(self) -> RoidmiCleaningSummary:
+        """Return information about cleaning runs."""
+        return RoidmiCleaningSummary(
+            {
+                prop["did"]: prop["value"] if prop["code"] == 0 else None
+                # max_properties limmit to 10 to avoid "Checksum error" messages from the device.
+                for prop in self.get_properties_for_mapping(max_properties=10)
+            }
+        )
+
+    @command()
+    def start(self) -> None:
+        """Start cleaning."""
+        return self.call_action_by(2, 1)
+
+    @command(click.argument("roomstr", type=str))
+    def start_room_sweep_unknown(self, roomstr: str) -> None:
+        """Start room cleaning.
+
+        FIXME: the syntax of room_sweep is unknown
+        """
+        return self.call_action_by(2, 3, roomstr)
+
+    @command(
+        click.argument("sweep_mode", type=EnumType(SweepMode)),
+        click.argument("clean_info", type=str),
+    )
+    def start_sweep_unknown(self, sweep_mode: SweepMode, clean_info: str) -> None:
+        """Start sweep with mode.
+
+        FIXME: the syntax of start sweep with mode is unknown
+        """
+        return self.call_action_by(14, 1, [sweep_mode.value, clean_info])
+
+    @command()
+    def stop(self) -> None:
+        """Stop cleaning."""
+        return self.call_action_by(2, 2)
+
+    @command()
+    def home(self) -> None:
+        """Return to home."""
+        return self.call_action_by(3, 1)
+
+    @command()
+    def identify(self) -> None:
+        """Locate the device (i am here)."""
+        return self.call_action_by(8, 1)
+
+    @command(click.argument("vol", type=int))
+    def set_sound_volume(self, vol: int):
+        """Set sound volume [0-100]."""
+        return self.set_property("volume", vol)
+
+    @command(click.argument("fanspeed_mode", type=EnumType(FanSpeed)))
+    def set_fanspeed(self, fanspeed_mode: FanSpeed):
+        """Set fan speed."""
+        return self.set_property("fanspeed_mode", fanspeed_mode.value)
+
+    @command(click.argument("sweep_type", type=EnumType(SweepType)))
+    def set_sweep_type(self, sweep_type: SweepType):
+        """Set sweep_type."""
+        return self.set_property("sweep_type", sweep_type.value)
+
+    @command(click.argument("path_mode", type=EnumType(PathMode)))
+    def set_path_mode(self, path_mode: PathMode):
+        """Set path_mode."""
+        return self.set_property("path_mode", path_mode.value)
+
+    @command(click.argument("work_station_freq", type=int))
+    def set_work_station_freq(self, work_station_freq: int):
+        """Set work_station_freq (2 means base dust colect every second time)."""
+        return self.set_property("work_station_freq", work_station_freq)
+
+    @command(click.argument("timing", type=str))
+    def set_timing_unknown(self, timing: str):
```

Comment:
```suggestion
    def set_timing(self, timing: str):
```
I think there is no need to describe this as unknown?